### PR TITLE
Chances

### PR DESCRIPTION
**Pull request by PedroBarge**
On Player add a end line in the bag
On Character add a enter
On Render add the name of the house
On Story correct spaces
On Game finalEnd done but need verifications

### CODE REVIEW

File: src/characters/Character.java
Comment (rodgreg02) [inline src/characters/Character.java:1]: why

File: src/characters/Player.java
Comment (rodgreg02) [inline src/characters/Player.java:1]: nice

File: src/gamecore/Game.java
Comment (rodgreg02) [inline src/gamecore/Game.java:1]: add .toLowerCase so it accepts both capitalized and non capitalized names

File: src/gamecore/Render.java
Comment (rodgreg02) [inline src/gamecore/Render.java:1]: good enough, needs grammar corrections

**Review by rodgreg02 — CHANGES_REQUESTED** [review]
add the .toLowerCase to class Game, so it can accept both capitalized and non capitalized guesses

**Comment by PedroBarge** [comment]
Update Menu added Rules option.
Update Game, know accept all lower or UPPER case.

**Review by rodgreg02 — APPROVED** [review]
ok ig

**Review by rodgreg02 — APPROVED** [review]
ok

**Pull request merged**